lib/streamqflite.dart

```diff
@@ -243,9 +243,10 @@ class QueryStream extends Stream<LazyQuery> {
 
   Stream<List<T>> mapToList<T>(T mapper(Map<String, dynamic> row)) {
     return _source.asyncMap((query) => query()).map((rows) {
-      var result = List<T>(rows.length);
+      //var result = List<T>(rows.length);
+      List<T> result = [];
```

Comment:
This removes the admittedly minor optimization of sizing the list so it doesn't have to reallocate, just curious if there's a way to do that now, maybe by defining a capacity?

Reply:
After searching might be able to use https://api.dart.dev/stable/2.16.2/dart-core/List/List.generate.html?